pkg/vault/test/foundry/QueryERC4626Buffer.t.sol

```diff
@@ -61,7 +55,7 @@ contract QueryERC4626BufferTest is BaseVaultTest {
     }
 
     function testSwapPreconditions() public view {
-        // bob should have the full boostedPool BPT.
+        // Bob should have the full boostedPool BPT.
```

Comment:
I believe boosted pool mentions are removed in another PR, right? (I've seen it in other files too, but I guess it's out of the scope of this one?)

Reply:
Yes, but that one's already merged. So if there are any left they're stragglers that we should address. Will fix.